packages/@ourworldindata/explorer/src/Explorer.tsx

```diff
@@ -859,7 +854,13 @@ export class Explorer
         this.setGrapherTable(
             BlankOwidTable(tableSlug, `Loading table '${tableSlug}'`)
         )
-        void this.futureGrapherTable.set(this.tableLoader.get(tableSlug))
```

Comment:
Does using `await` here instead of `void` not solve it? I find the `PromiseSwitcher` abstraction mind-bending, but it seems useful, and we lose that functionality (handling [out of order arrival of network requests](https://react.dev/reference/react/useEffect#fetching-data-with-effects) when switching views).

Not sure how to get the same functionality without it in the current implementation, but we should solve it as it is a common problem when fetching data.